app/config/permissions.js

```diff
@@ -122,6 +122,23 @@ const groups = [
       'edit-in-treatment-submissions',
       'create-subcases-from-submissions',
       'delete-submissions',
+    ],
+    readOnlyPermissions: [
+      'view-all-ongoing-signatures',
+      'search-publication-flows',
```

Comment:
Not a full fan of the duplications.
Could we:
- create 2 lists
- 1 with all reading permissions
- 1 with all writing permissions
- if not mobile or edge > combine the lists